src/schema/v2/order/__tests__/MeOrder.test.ts

```diff
@@ -348,5 +352,207 @@ describe("Me", () => {
       })
     })
+
+    describe("fulfillmentDetails", () => {
+      let consoleErrorSpy
+      beforeEach(() => {
+        const realConsoleError = console.error
+        consoleErrorSpy = jest.spyOn(console, "error").mockImplementation()
+        console.error = (...args) => {
+          if (args[0] !== "Parse phone number error: ") {
+            realConsoleError(...args)
+          }
+        }
+      })
+      afterEach(() => {
+        consoleErrorSpy.mockRestore()
+      })
+
+      it("returns phoneNumber if there is a blank country code", async () => {
+        orderJson.buyer_phone_number_country_code = ""
+        orderJson.buyer_phone_number = "7738675309"
+        context = {
+          meLoader: jest.fn().mockResolvedValue({ id: "me-id" }),
+          meOrderLoader: jest.fn().mockResolvedValue(orderJson),
+          artworkLoader: jest.fn().mockResolvedValue(artwork),
+          authenticatedArtworkVersionLoader: jest
+            .fn()
+            .mockResolvedValue(artworkVersion),
+        }
+        const query = gql`
+          query {
+            me {
+              order(id: "order-id") {
+                fulfillmentDetails {
+                  phoneNumber {
+                    countryCode
+                    regionCode
+                    originalNumber
+                    isValid
+                    display
+                  }
+                }
+              }
+            }
+          }
+        `
+        const result = await runAuthenticatedQuery(query, context)
+        expect(result.me.order.fulfillmentDetails.phoneNumber).toEqual({
+          countryCode: null,
+          regionCode: null,
+          originalNumber: "7738675309",
+          isValid: false,
+          display: null,
+        })
+      })
+
+      it("returns only countryCode, originalNumber and a null display field if validation fails", async () => {
+        orderJson.buyer_phone_number_country_code = "us"
+        orderJson.buyer_phone_number = "7738asdf675309"
+        context = {
+          meLoader: jest.fn().mockResolvedValue({ id: "me-id" }),
+          meOrderLoader: jest.fn().mockResolvedValue(orderJson),
+          artworkLoader: jest.fn().mockResolvedValue(artwork),
+          authenticatedArtworkVersionLoader: jest
+            .fn()
+            .mockResolvedValue(artworkVersion),
+        }
+        const query = gql`
+          query {
+            me {
+              order(id: "order-id") {
+                fulfillmentDetails {
+                  phoneNumber {
+                    countryCode
+                    regionCode
+                    originalNumber
+                    isValid
+                    display(format: E164)
+                  }
+                }
+              }
+            }
+          }
+        `
+        const result = await runAuthenticatedQuery(query, context)
+        expect(result.me.order.fulfillmentDetails.phoneNumber).toEqual({
+          countryCode: "1",
+          regionCode: null,
```

Comment:
Is it confusing that the persisted "us" is not returned in `regionCode` but "1" is returned in the `countryCode`?

Reply:
yes... yes it is.

Reply:
yes. I think this was actually the library not working as expected. it seems like if the library 

Good number, the backing code works:
![image](https://github.com/user-attachments/assets/f238a5c1-321b-471f-b59e-93fab9ad77de)

Bad number it doesn't - unless we use the country code (+1) to get the region code ("us")
![image](https://github.com/user-attachments/assets/08d4dc22-2ce5-48e6-886e-1ae968d68ba2)
![image](https://github.com/user-attachments/assets/0b5fc228-4426-45d6-a3b9-e4f625c0c2e2)